### PR TITLE
feat: added button component

### LINKED ISSUE #3: Add Button Component
self explanatory

### PR DESCRIPTION
**Pull request by shraddha15-creator**
Added Button component
- Primary button
- Button with link
- Rounded buttons

btn-component

### CODE REVIEW

File: components/components.html [hunk 1]
```diff
@@ -95,8 +95,40 @@ <h2>Dot Badge</h2>
...
+            <div>
+                <a href="#" class="btn btn-link">Google.com</a>
+                <a href="#" class="btn btn-link">Read More..</a>
+                <a href="#" class="btn btn-link">Go to next page</a>
+                
+            </div>
+        </div>
+        <iframe
+  src="https://carbon.now.sh/embed/ZjU3bqueLdKQikyfD1jM"
+  style="width: 930px; height: 564px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (imabhinavagarwal) [inline components/components.html]: Please use this style in iframe to be in sync with rest
style="border:0; transform: scale(1); overflow:hidden;"

Also, please fix the indentation

File: components/components.html [hunk 2]
```diff
@@ -95,8 +95,40 @@ <h2>Dot Badge</h2>
...
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
     </section>
+    <hr>
+    </body>
```
Comment (imabhinavagarwal) [inline components/components.html]: Remove this tag from here. Body tag is closed at end of file after all the sections

**Review by imabhinavagarwal — CHANGES_REQUESTED** [review]
Please make changes as per the comment. Also, please help to update the Readme file with components and sub components you are adding
issue #3

File: components/components.html (hunk 1 above)
Comment (imabhinavagarwal) [reply components/components.html]: @shraddha15-creator By fix the indentation, I meant just for the iframe tag 😅

**Comment by imabhinavagarwal** [comment]
@shraddha15-creator Please resolve conflicts

**Review by imabhinavagarwal — APPROVED** [review]
There was duplicated code, just fixed that. Rest was okay

**Comment by shraddha15-creator** [comment]
> There was duplicated code, just fixed that. Rest was okay

why did you not accept this pr for hacktoberfest?

**Comment by imabhinavagarwal** [comment]
> > There was duplicated code, just fixed that. Rest was okay
> 
> why did you not accept this pr for hacktoberfest?

@shraddha15-creator I have marked it as hacktoberfest-accepted

**Pull request merged**